packages/e2e-tests/tests/basic-functionality.spec.ts

```diff
@@ -151,6 +151,48 @@ test('send message', async ({ page }) => {
   await expect(receivedMessageText).toHaveText(messageText)
 })
 
+test('edit message', async ({ page }) => {
+  const userA = getUser(0)
```

Comment:
Very nice!

Reply:
Yep! It was a pleasure. Thanks for setting up the framework!